lib/src/state_machine.dart

```diff
@@ -117,3 +135,53 @@ class State<StateIdentifier> {
   /// and [actions] associated with that state.
   State(this.identifier, {required this.events, required this.actions});
 }
+
+/// A state diagram generator for FSM.
+///
+/// Outputs to vcd format at [outputPath].
+class MermaidStateDiagram {
```

Comment:
Should this be a private class or do you intend to expose this as a public api?
`_MermaidStateDiagram`

Reply:
Ya, this should be private class instead.